### PR TITLE
Propolsal for MAINTAINERS.md and GOVERNANCE.md

### PR DESCRIPTION
**Pull request by OrlinVasilev**
As I'm still not official member this is just a propolsal to stream line the repository acording to CNCF standarts

Needed updates per file:
MAINTAINERS.md
- add real names for each maintainer
- add company affiliation

GOVERNANCE.md
- maintainers team in github needs update - @external-secrets/maintainers
- ESO dev mailing list needs update - ESO dev mailing

Propolsal template is optional but at some point is good to have a formal process for new features!

Orlix

Signed-off-by: OrlinVasilev <ovasilev@vmware.com>

### CODE REVIEW

File: GOVERNANCE.md
Comment (knelasevero) [inline GOVERNANCE.md:79]: I think this part need some tweaking to include the already design/proposal process that we already have in https://github.com/external-secrets/external-secrets/tree/main/design

But we can combine what is being described here and what we already do

File: MAINTAINERS.md
Comment (knelasevero) [inline MAINTAINERS.md:18]:
```suggestion
## External Secrets Operator Core Contributors & Stakeholders
```

File: GOVERNANCE.md
Comment (OrlinVasilev) [reply GOVERNANCE.md:79]: Will try to fix that tomorrow ! Thanks

**Comment by knelasevero** [comment]
@OrlinVasilev we have our bot configuration file that also holds this kind of information.

https://github.com/external-secrets/external-secrets/blob/main/.github/PAUL.yaml

I'm a bit worried about the redundancy and having to update both files.

@gusfcarvalho @moolen what do you think?

**Comment by OrlinVasilev** [comment]
@knelasevero , not saying you don't have it I took it from there :D 
But the project now is under CNCF so is good to follow the CNCF guides and practices e.g. 
https://clomonitor.io/projects/cncf/external-secrets
just search for maintainers and you will get the idea 

other resource is https://github.com/cncf/project-template

Hope that helps!

**Comment by OrlinVasilev** [comment]
or maybe we can do maintainers.md to point to the paul.yaml as reference and single point of truth!

**Comment by knelasevero** [comment]
I see! Yeah, if we can have a way to avoid having to update both files, that would be great! Or maybe have a Makefile command that keeps files in sync and maybe CI checking that.

But I don't want to block the PR now with this nit :)

LGTM, I will let @moolen and @gusfcarvalho have a look as well

File: GOVERNANCE.md
Comment (moolen) [inline GOVERNANCE.md:64]:
```suggestion
`design` folder in the [external-secrets repository](https://github.com/external-secrets/external-secrets/tree/main/design)
```

File: GOVERNANCE.md
Comment (moolen) [inline GOVERNANCE.md:67]:
```suggestion
`000-clear-old-tags-with-policies.md`.
```

File: GOVERNANCE.md
Comment (moolen) [inline GOVERNANCE.md:63]:
```suggestion
The proposal should be documented as a separated markdown file pushed to the
```

**Review by moolen — COMMENTED** [review]
Hey @orlinVasilev, thanks for bringing this up! I didn't find the time earlier to look at this :eyes: 
This looks very good. Just a few typos, otherwise LGTM :+1:

**Comment by OrlinVasilev** [comment]
@moolen thank you!

**Comment by gusfcarvalho** [comment]
LGTM :)

**Comment by OrlinVasilev** [comment]
Thanks @gusfcarvalho @knelasevero @moolen :)

**Review by moolen — APPROVED** [review]
LGTM :+1: 
just rebased onto master to fix issue with ubi image.

**Comment by moolen** [comment]
gonna merge this then with a supermajority vote :tada: :wink:

**Comment by OrlinVasilev** [comment]
Thank you folks super happy to contribute! Can I have "hacktoberfest-accepted" label as well so we can show that the repository was full on hacktoberfest!

**Pull request merged**
